app/assets/javascripts/admin/models/admin-user.js.es6

```diff
@@ -168,6 +168,17 @@ const AdminUser = Discourse.User.extend({
     }).catch(popupAjaxError);
   },
 
+  disableSecondFactor() {
+    const self = this;
+    return ajax("/admin/users/" + this.get('id') + "/disable_second_factor", {
+      type: 'PUT'
+    }).then(function() {
+      self.setProperties({
```

Comment:
If you're only updating one property it's preferable to use `this.set('second_factor_enabled', true)` rather than `setProperties`